README.md

```diff
@@ -41,44 +21,118 @@ This allows you to check in any changes you make to your private copy of the sol
     |- pipeline               - automated testing pipeline source files
 ```
 
-## Deploy the Solution From Github
+# Deploying the Solution
```

Comment:
Usually a top-level heading is used for the title of the document

Reply:
usually, though the layout of the readme currently has 2 sections that are arguably whole documents in themselves (deploying and testing) so that's why they are H1 level

Reply:
The title of the readme is the title of the solution. I would treat deploying and testing as second-level headings since they are both releated to the solution